### PR TITLE
Integrate Travis CI and Coveralls to the Repository

### PR DESCRIPTION
**Pull request by sarthakgoel1997**
## Description
- Add `.travis.yml` config file: Have currently commented the deploy flag
- Add `.gitignore` file
- Add `.coveralls.yml` to report code coverage
- Update `README.md` file to display build and coverage badges
- Push `healthScore/.elasticbeanstalk/config.yml` file
- Fix all **black** and **flake8** linter issues for the build to pass

## How did you test the changes?
- Tested the changes by checking the Travis CI and Coveralls dashboards

### Travis CI Dashboard
image

### Coveralls Dashboard
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+healthScore/.venv/bin/pip3
```
Comment (shreyas2499) [inline .gitignore]: Please remove this file

File: .coveralls.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+repo_token: 9eh2ULMq3sJxydGLViI1OaEcxqbc0vRL7
```
Comment (shreyas2499) [inline .coveralls.yml]: Can you add these values to the travis environment variables?

File: healthScore/requirements.txt [hunk 3]
```diff
@@ -1,21 +1,35 @@
...
 distlib==0.3.8
 Django==4.2
+docopt==0.6.2
 filelock==3.13.1
+flake8==7.0.0
 idna==3.6
 jmespath==1.0.1
+mccabe==0.7.0
+mypy-extensions==1.0.0
+mysql-connector==2.2.9
```
Comment (shreyas2499) [inline healthScore/requirements.txt]: Please remove the mysql related packages as they might cause deployment issues

Comment (sarthakgoel1997) [reply healthScore/requirements.txt]: Right, updating the PR

File: .coveralls.yml (hunk 2 above)
Comment (sarthakgoel1997) [reply .coveralls.yml]: Sure, let's give this a try!

File: .gitignore (hunk 1 above)
Comment (sarthakgoel1997) [reply .gitignore]: Yes, done

**Review by shreyas2499 — APPROVED** [review]

**Review by shreyas2499 — APPROVED** [review]

**Pull request merged**